sherpa/models/tests/test_template.py

```diff
@@ -134,16 +142,15 @@ def tearDown(self):
         self.model = None
 
     def test_template_model_evaluation(self):
-        self.model.thawedpars = [0,1,0,1]
+        self.model.thawedpars = [0, 1, 0, 1]
         # We want to evaluate the model, but do not check the result
         self.model(self.x)
 
 #    def test_template_query_index(self):
 #        expected = 5
 #        result = self.model.query_index([0,1,0,1])
-#        self.assertEqual(expected, result,
-#                         "Expected %s instead of %s" % (str(expected), str(result)))
-
+#        self.assertEqual(expected, result)
+#
```

Comment:
it would have helped, at the time, adding a couple of comments as to why these tests were commented out. Have you tried running them?